website/content/docs/secrets/ssh/one-time-ssh-passwords.mdx

```diff
@@ -1,16 +1,14 @@
 ---
 layout: docs
-page_title: One-Time SSH Passwords (OTP) - SSH - Secrets Engines
-description: |-
-  The One-Time SSH Password (OTP) SSH secrets engine type allows a Vault server
-  to issue a One-Time Password every time a client wants to SSH into a remote
-  host using a helper command on the remote host to perform verification.
+page_title: One-time SSH passwords
+description: >-
+  Issue a one-time password every time a client connects to a remote machine via SSH using a helper command on the host to perform verification.
 ---
 
-# One-Time SSH passwords
+# One-time SSH passwords
 
 The One-Time SSH Password (OTP) SSH secrets engine type allows a Vault server to
-issue a One-Time Password every time a client wants to SSH into a remote host
+issue a one-time password every time a client wants to SSH into a remote host
```

Comment:
This is capitalized to reflect that OTP is an acronym for this.

Reply:
It's part of a sentence talking about generating a one-time password. 

Do you prefer it to use the acrynym? --> `...issue an OTP every time a client...`

Reply:
No, actually that seems fine.